### ISSUE TITLE
[CS 1.6] Fix duplication messages in console

### ISSUE THREAD

**Issue by eat1k** [ValveSoftware/halflife#2159]
The messages in console are duplicated. It happens only if we start a game via "New Game".

![console](https://user-images.githubusercontent.com/31934972/56459478-83429800-6394-11e9-8678-484cee682513.jpg)

**Comment by shelru** [comment]
Ну ведь у тебя одновременно и серверная часть и клиентская.
Запусти hlds отдельно и зайди на локальный сервер.

**Comment by eat1k** [comment]
> Ну ведь у тебя одновременно и серверная часть и клиентская.
> Запусти hlds отдельно и зайди на локальный сервер.

Yes, in a normal server it works, but if it's can be fixed it would be nice.

**Comment by shelru** [comment]
Собственно говоря оно так и должно работать. Ты ведь можешь хостить у себя игроков, запусти две кс одновременно и зайди с одной на сервер созданный в другой, одна будет клиентом и хостом, другая клиентом.
Это обычный SERVER_PRINT, здесь ничего страшного нет.
Со второй кс напиши что нибудь в чат, тебе сразу же отобразится в консоли первой с различным количеством пробелов, после того как сработает худ скроллинг тебе в первой кс отобразится с обычным количеством пробелов то же сообщение, при этом во второй кс по истечению худ скролла отобразит только 1 сообщение.

Здесь нет бага, это правильная работа.

**Comment by Mistrick** [comment]
anyway there should not be duplicates, this adds confusion to listen server hoster

**Comment by shelru** [comment]
Only one bug there is \n\n, it's bad.

**Comment by 2010kohtep** [comment]
A similar problem exists on servers that use chat modifiers. Some plugins send ```SayText``` and ```TextMsg``` (if memory serves me) at the same time, which creates duplicate messages. Perhaps this is done to fix #2060 by the server side.

**Comment by shelru** [comment]
Но опять же, с появлением фикса с русским чатом в консоли - это станет неактуальным. В основном это проблема серверов где преобладает большая часть нонстим игроков(хотя...), сложно сказать, я вот плагины чата всегда писал и дублировал в консоль лишь стим клиентам и только при наличии не латинских символов в тексте.

**Comment by XakepSDK** [comment]
Please, english only

**Comment by shelru** [comment]
Important messages were written in english.
